src/components/pacbio/PacbioTubeSelectedList.vue

```diff
@@ -48,13 +48,13 @@ export default {
     }
   },
   computed: {
-    ...mapGetters(['selectedTubes', 'wellList', 'requestList']),
+    ...mapGetters(['poolItem', 'tubeItem', 'selectedRequests']),
     selectedTubeRequests() {
+      return this.selectedRequests
```

Comment:
I think this will show all requests from both plates and tubes. Where as for this particular table we only want to show the requests from tubes.